packages/files-ui/src/Components/Modules/FileBrowsers/SharedFileBrowser.tsx

```diff
@@ -87,6 +88,9 @@ const SharedFileBrowser = () => {
       }
     })
   }), [arrayOfPaths, bucket, redirect])
+  const currentFolder = useMemo(() => {
+    return arrayOfPaths[arrayOfPaths.length - 1]
```

Comment:
should we go safe here and make sure this never end-up being `arrayOfPaths[-1]`?
```suggestion
    return !!arrayOfPaths.length && arrayOfPaths[arrayOfPaths.length - 1]
```